package.json

```
@@ -12,23 +12,26 @@
     "access": "public"
   },
   "scripts": {
-    "start": "j1-integration collect",
+    "start": "NODE_TLS_REJECT_UNAUTHORIZED=0 j1-integration collect",
     "graph": "j1-integration visualize",
     "lint": "eslint . --cache --fix --ext .ts,.tsx",
     "format": "prettier --write '**/*.{ts,js,json,css,md,yml}'",
     "type-check": "tsc",
-    "test": "jest",
+    "test": "NODE_TLS_REJECT_UNAUTHORIZED=0 jest",
```

This caused a failure in CI tests on the windows machine: https://github.com/JupiterOne/graph-rapid7/pull/1/checks?check_run_id=1182455250

Would the following work? I assume that you only need the `NODE_TLS_REJECT_UNAUTHORIZED` when making actual HTTP calls, and that the recordings will work fine without that flag, but I could be wrong about that.

```suggestion
    "test:local": "NODE_TLS_REJECT_UNAUTHORIZED=0 && yarn test",
    "test": "jest",
```

---

That did work, good suggestion! :+1: